macros/upload_individual_datasets/upload_test_executions.sql

```diff
@@ -148,3 +148,56 @@
         {{ return("") }}
     {% endif %}
 {% endmacro -%}
+
+{% macro snowflake_get_test_executions_dml_sql(tests) -%}
```

Comment:
```suggestion
{% macro snowflake__get_test_executions_dml_sql(tests) -%}
```
Need to add another underscore